scalar/uuid_test.go

```diff
@@ -59,6 +63,8 @@ func TestUUIDSet(t *testing.T) {
 		},
 		{source: nilPointerByteArray, result: UUID{}},
 		{source: nilPointerString, result: UUID{}},
+		{source: &uuid.UUID{}, result: UUID{Value: [16]byte{0}, Valid: true}},
+		{source: uuid.UUID{}, result: UUID{Value: [16]byte{0}, Valid: true}},
```

Comment:
Please add the `uuid.DNSNamespace` case as well (ass `uuid.Nil` is trivial only)